charts/incubator/youtrack/values.yaml

```
@@ -0,0 +1,41 @@
+image:
+  repository: jetbrains/youtrack
+  tag: 2022.2.55618
+  pullPolicy: IfNotPresent
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: true
+
+podSecurityContext:
+  runAsGroup: 13001
+  runAsUser: 13001
+  fsGroup: 13001
+
+service:
+  main:
+    ports:
+      main:
+        port: 8080
```

```suggestion
        port: 18080
```

Let's not have a common port here, as it most likely will conflict with other apps